tests/end_to_end/test_kakarot.py

```diff
@@ -393,3 +393,40 @@ async def test_should_raise_when_tx_view_entrypoint(self, kakarot, entrypoint):
             receipt = await RPC_CLIENT.get_transaction_receipt(tx_hash)
             assert receipt.execution_status.name == "REVERTED"
             assert "Only view call" in receipt.revert_reason
+
+    class TestEthRPCEntrypoints:
+        async def test_should_return_native_balance_of(self, kakarot, new_eoa):
+            eoa = await new_eoa()
+            balance = (
+                await kakarot.functions["eth_get_balance_of"].call(
+                    evm_address=int(eoa.address, 16)
+                )
+            ).balance
+            assert balance == 50000000000000000000
+
+        async def test_should_return_transaction_count(self, kakarot, new_eoa):
+            eoa = await new_eoa()
+            tx_count = (
+                await kakarot.functions["eth_get_transaction_count"].call(
+                    evm_address=int(eoa.address, 16)
+                )
+            ).tx_count
+            assert tx_count == 0
+
+            await invoke(
+                "kakarot",
+                "write_account_nonce",
+                int(eoa.address, 16),
+                1,
+            )
```

Comment:
```suggestion
            await invoke(
                "kakarot",
                "write_account_nonce",
                int(eoa.address, 16),
                1
            )
```

and I think that this fits in one line